src/pages/base.ts

```diff
@@ -1,5 +1,5 @@
 import config from '../core/config.helper';
-import { waitForInvisibilityOf, waitForVisibilityOf } from '../web/cucumber/wait-for-condition.helper';
+import { waitForInvisibilityOf, waitForVisibilityOf } from '../web/methods/wait-for-condition.helper';
```

Comment:
Improve the naming, e.g. `wait-for-condition.methods.ts` so we know where to find

Reply:
For now i don't want to change it in base.ts and nameing as I am thinking to include this inside kakMethods, and it will require more changes

Reply:
Will change name to wait-for-condition.methods

Reply:
But in future i am thinking of putting this methods inside `methods` export